### DISCUSSION TITLE
Matematik Form 1 bab 6 : Persamaan Linear

### DISCUSSION THREAD

**Discussion by E-Pendidikan-Stem** [E-Pendidikan-Stem/Matematik#12] [Ideas]
Adakah teori persamaan linear dapat di aplikasikan dalam kehidupan seharian kita?

**Comment by Botman508** [comment]
ya,kerana jika diperhatikan dan diteliti apa yang berlaku dalam kehidupan seharian kita ada terkait rapat dengan sistem persamaan linear terutamanya dalam bidang sains,industri dan juga permasalahan ekonomi.

**Reply by E-Pendidikan-Stem** [reply]
Contohnya?

**Reply by Botman508** [reply]
contohnya dalam sistem pencukaian perkhidmatan

**Reply by E-Pendidikan-Stem** [reply]
Benar 😊👍 Tahniah anda betul ⭐
